parser-typechecker/src/Unison/TermParser.hs

```diff
@@ -324,7 +324,7 @@ boolean = ((\t -> Term.boolean (ann t) True) <$> reserved "true") <|>
           ((\t -> Term.boolean (ann t) False) <$> reserved "false")
 
 seq :: Var v => TermP v -> TermP v
-seq = Parser.seq Term.seq
+seq = Parser.seq Term.list
```

Comment:
You could switch this one to `list` too.

Reply:
No problem, sure I'll do this in an hour or so. Thanks!